Woof/Woof/View/SitterFlow/SitterOnboardingView.swift

```diff
@@ -0,0 +1,53 @@
+import SwiftUI
+
+/// A view to onboard the sitter by filling in the mandatory information about themselves.
+struct SitterOnboardingView: View {
+    /// View model responsible to manage data from model layer
```

Comment:
What `SitterProfileViewModel`'s functionality do you want to change? Or what functionality do you want to add? 
```suggestion
struct SitterOnboardingView: View {
    typealias SitterOnboardingViewModel = SitterProfileViewModel
    /// View model responsible to manage data from model layer
```

Reply:
Yes, you're right, the functionality of the current view model will be used, and nothing new is needed.

Reply:
WDYT about to add type alias in my suggestion above:
`typealias SitterOnboardingViewModel = SitterProfileViewModel`